bundles/org.openhab.binding.icalendar/src/main/java/org/openhab/binding/icalendar/internal/logic/Event.java

```diff
@@ -66,12 +71,34 @@ public boolean equals(@Nullable Object other) {
             return false;
         }
         final Event otherEvent = (Event) other;
-        return (this.title.equals(otherEvent.title) && this.start.equals(otherEvent.start)
-                && this.end.equals(otherEvent.end));
+        String title = this.title;
+        return (this.start.equals(otherEvent.start) && this.end.equals(otherEvent.end)
```

Comment:
`equals()` should also include the added fields `description`, `location`, `comment` and `contact`. I would also recommend a corresponding implementation of `hashCode` to ensure the [general contract](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Object.html#equals(java.lang.Object)).

Reply:
OK